docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -161,12 +164,12 @@ quarkus.oidc.auth-server-url=http://localhost:8180/realms/quarkus/
 quarkus.oidc.client-id=quarkus-app
 
 # This is a key which will be used to retrieve a secret from the map of credentials returned from CredentialsProvider
-quarkus.oidc.credentials.jwt.secret-provider.key=mysecret-key
+quarkus.oidc.credentials.jwt.secret-provider.key=mysecret-key.
```

Comment:
```suggestion
quarkus.oidc.credentials.jwt.secret-provider.key=mysecret-key
```

Reply:
Removed period again per your suggestion. Thanks for clarifying this.